### DISCUSSION TITLE
difference between wet test driver and wintun

### DISCUSSION THREAD

**Discussion by hyfand** [doronz88/pymobiledevice3#1028] [Q&A]
May I ask if you still need to install Tencent's wetest driver if you want to use 17 devices before 17.4? Can the newly used Wintun support versions before 17.4? I still don't quite understand the difference between wet test driver and wintun. May I ask for your advice.

**Comment by doronz88** [comment]
When working with iOS 17.0-17.3.1 devices you'll require driver to communicate with the device.
The `wintun` module should only wrap APIs that from my understanding should already exist on every windows workstation via the wintun driver (I hope so)

https://www.wintun.net/
